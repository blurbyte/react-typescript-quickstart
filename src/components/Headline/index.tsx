import styled from '@styles/styledComponents';
import { Color } from '@styles/colors';

const Headline = styled.h1`
  font-size: 5rem;
  font-weight: 300;
  letter-spacing: -0.2rem;
  color: ${Color.Blue};
  border-top: 2px solid ${Color.Gray};
  padding-top: 1rem;
  margin-top: 3rem;
  margin-bottom: 2.4rem;
  line-height: 1.2;
`;

export default Headline;
