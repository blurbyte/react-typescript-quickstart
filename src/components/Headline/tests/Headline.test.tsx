import * as React from 'react';
import { shallow } from 'enzyme';

import Headline from '../index';

describe('Headline', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Headline />);
    expect(wrapper).toMatchSnapshot();
  });
});
