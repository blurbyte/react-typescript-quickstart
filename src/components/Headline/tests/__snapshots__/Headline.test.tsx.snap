// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headline renders correctly 1`] = `
.c0 {
  font-size: 5rem;
  font-weight: 300;
  -webkit-letter-spacing: -0.2rem;
  -moz-letter-spacing: -0.2rem;
  -ms-letter-spacing: -0.2rem;
  letter-spacing: -0.2rem;
  color: #4a6cbb;
  border-top: 2px solid #908e89;
  padding-top: 1rem;
  margin-top: 1rem;
  margin-bottom: 2.4rem;
  line-height: 1.2;
}

<h1
  className="c0"
/>
`;
